js/components/tabs.js

```diff
@@ -27,6 +27,25 @@ class Tabs extends ImmutableComponent {
     this.onNextPage = this.onNextPage.bind(this)
     this.onNewTabLongPress = this.onNewTabLongPress.bind(this)
     this.wasNewTabClicked = this.wasNewTabClicked.bind(this)
+    this.onMouseLeave = this.onMouseLeave.bind(this)
+    this.onFrameClose = this.onFrameClose.bind(this)
+
+    this.state = {
+      fixTabWidth: null
+    }
+  }
+
+  onMouseLeave () {
+    this.setState({
+      fixTabWidth: null
+    })
+    this.forceUpdate()
```

Comment:
Why I asked was because `onMouseLeave` event is very generic looking from a appState perspective. 
I can make it specific to tab like `onMouseLeaveTab`, etc